R/summaries.R

```diff
@@ -258,11 +265,23 @@ setMethod("countDominantTaxa", signature = c(x = "SummarizedExperiment"),
                      call. = FALSE)
             }
         }
+        # name check
+        if(!is.null(name)) {
+          if(is.character(name)==F) {
```

Comment:
Change to --> 

if( is.character(name) && length(name) == 1 )

Always use FALSE and not F. F is just a variable that has default value of FALSE (user can change the value of F to something else and then problems occur).